client/remote_access.go

```diff
@@ -39,7 +39,7 @@ type RemoteAccessTarget struct {
 
 	// LocalPort is the port on the local machine to which the remote port is forwarded.
 	// If set to 0, a random port will be chosen.
-	LocalPort uint16
+	LocalPort string
```

Comment:
Should we make `RemotePort` a string as well - I think consistency would be good.

Also, I think we need to update the comment to remove "If set to 0, a random port will be chosen." as this is currently not the case (and we probably won't have like this). Currently, setting local port to 0 will reuse the remote port.